diff -ur ZoneAdmin-0.2-beta1/includes/smarty/libs/Smarty.class.php ZoneAdmin-0.2-beta1.new/includes/smarty/libs/Smarty.class.php
--- ZoneAdmin-0.2-beta1/includes/smarty/libs/Smarty.class.php	2007-09-27 12:12:39.000000000 +0200
+++ ZoneAdmin-0.2-beta1.new/includes/smarty/libs/Smarty.class.php	2007-10-12 22:10:07.958967866 +0200
@@ -79,7 +79,7 @@
      *
      * @var string
      */
-    var $compile_dir     =  'templates_c';
+    var $compile_dir     =  '/var/lib/ZoneAdmin/templates_c';
 
     /**
      * The directory where config files are located.
Tylko w ZoneAdmin-0.2-beta1.new/includes/smarty/libs: Smarty.class.php~
diff -ur ZoneAdmin-0.2-beta1/index.php ZoneAdmin-0.2-beta1.new/index.php
--- ZoneAdmin-0.2-beta1/index.php	2007-09-27 12:12:39.000000000 +0200
+++ ZoneAdmin-0.2-beta1.new/index.php	2007-10-12 22:03:29.959241829 +0200
@@ -39,7 +39,7 @@
 $tpl = new Smarty;
 
 //smarty plugins laden (im moment nur eine substr() funktion)
-$tpl->plugins_dir[] = "./includes/smarty_plugins";
+$tpl->plugins_dir[] = "/usr/share/php/Smarty/plugins";
 
 //language basis file laden
 $tpl->config_load(getcwd().'/lang/lang_en.conf','en');
Tylko w ZoneAdmin-0.2-beta1.new/: index.php~
